services/wildfly/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+APP_NAME="wildfly"
```

Comment:
i2b2-wildfly with the real upstream i2b2 war version would make it easier to understand what it's good for.